assets/cubyz/biomes/savannah/savannah.zig.zon

```diff
@@ -0,0 +1,65 @@
+.{
```

Comment:
The file name is now redundant `savannah/savannah` and `savannah/elevated_savannah`.
I'd prefer something like `savannah/base` and `savannah/elevated`